tree/ntuple/v7/doc/specifications.md

```diff
@@ -1054,27 +1062,51 @@ The limits refer to a single RNTuple and do not consider combinations/joins such
 
 ## Glossary
 
-TODO: This glossary is not yet complete
+### Anchor
+
+The anchor is a data block that makes the entry point to an RNTuple.
+The anchor is specific to the RNTuple container in which the RNTuple data are embedded (e.g., a ROOT file).
+The anchor must provide the information to load the header and the footer **envelopes**.
+
+### Cluster
+
+A cluster is a set of **pages** that contain all the data belonging to a certain entry range.
+The data set is partitioned in clusters.
+A typical cluster size is tens to hundreds of megabytes.
 
 ### Column
 
 A column is a storage backed vector of a number of **elements** of a simple type.
```

Comment:
Probably good enough even-though it is semantically not quite true anymore (Streamed fields) (it is still technically true since it is backed by a bag of bytes ...)